src/types/modules/shortid.d.ts

```diff
@@ -0,0 +1 @@
+declare module "shortid";
```

Comment:
Welp, this is not the way it is usually done.

First of all, shortid is deprecated, and they recommend using nanoid on [their npm page](https://www.npmjs.com/package/shortid)

Secondly, shortid has external type definitions at `@types/shortid` (remember the screenshots of the npm page I've attached earlier?). As such it is much better to rely on community typings, than rolling out your own stubs (unless community typings are missing, outdated, or god awful)